lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -545,6 +577,28 @@ sub save_edit :Chained('base') :PathPart('save') :Args(0) {
     return $c->forward($c->view('JSON'));
 }
 
+sub usercheck :Chained('base') :PathPart('usercheck') :Args() {
+    my ( $self, $c ) = @_;
+
+    my $username = $c->req->params->{username};
+    my $type = $c->req->params->{type};
```

Comment:
Should probably fall back to `''` just in case there are no params. Accessing the URL gives me `Use of uninitialized value $type in string eq at /home/ubuntu/community-platform/script/../lib/DDGC/Web/Controller/InstantAnswer.pm line 588.`.

```perl
my $username = $c->req->params->{username} || '';
my $type = $c->req->params->{type} || '';
```

would probably be better.